src/uninstal.c

```diff
@@ -321,8 +321,8 @@ main(int argc, char *argv[])
     int		i;
     struct stat st;
     char	icon[BUFSIZE];
-    char	path[BUFSIZE];
-    char	popup_path[BUFSIZE];
+    char	path[MAX_PATH];
+    char	popup_path[MAX_PATH];
```

Comment:
`popup_path` is passed to `popup_gvim_path()`, and in the line 67 in the function, `BUFSIZE` is used.
The line 67 should be also changed to `MAX_PATH`.